### ISSUE TITLE
[AVT] Tabs does not have unique labels specified with aria-label or aria-labelledby

### ISSUE THREAD

**Issue by yanweibj** [carbon-design-system/carbon#5403]
## Environment

> Operating system
Mac OS
> Browser
Chrome
> Automated testing tool and ruleset
DAP
IBM Accessibility January 2020
IBM Accessibility Experimental

## Detailed description

> What version of the Carbon Design System are you using?
Carbon v10
> What did you expect to happen?
Tabs must have unique labels specified with aria-label or aria-labelledby
> What happened instead?
Tabs must have unique labels specified with aria-label or aria-labelledby
https://aat.w3ibm.mybluemix.net/token/a1f74e74-a651-4035-ab2f-16b524d25c84/f4811aea-af41-43c1-b82f-61d78aec22c5/archives/2019SeptDeploy//doc/w3/help/en-US/idhi_accessibility_check_g1164.html
> What WCAG 2.1 checkpoint does the issue violate?
2.4.1 Bypass Blocks
## Steps to reproduce the issue

1.   Render Tabs component
2.  In inspect check that accessible name does not have unique labels specified with aria-label or aria-labelledby

**Comment by yanweibj** [comment]
<img width="671" alt="image" src="https://user-images.githubusercontent.com/16932498/75252169-10594780-5817-11ea-82b9-6bec414d74a4.png">

**Comment by abbeyhrt** [comment]
Hi @yanweibj! 👋 Thank you for making this issue! Since this is only an issue when there are multiple navigation roles on a page, this might not be something we address on our side. I will look into this to see what the best solution is. In the meantime, you can resolve this on your end by adding an `aria-label` with your own unique label on the `Tabs` component. Here's an [example]( https://codesandbox.io/s/beautiful-colden-0u5bg?fontsize=14&hidenavigation=1&theme=dark)

That should address the violation. Let me know if that temporary solution doesn't work and we can figure something else out!

**Comment by yanweibj** [comment]
@abbeyhrt Thanks for your help, we will try to fix this problem by adding an aril-label.

**Comment by abbeyhrt** [comment]
Great! Closing this issue for now, feel free to `@` me if it needs to be reopened!

**Comment by carmacleod** [comment]
Unfortunately, adding aria-label is the wrong fix. The right fix is to delete role="navigation". Please see: https://github.com/carbon-design-system/carbon/issues/6931
